apisix/plugins/authz-keycloak.lua

```diff
@@ -695,8 +703,107 @@ local function fetch_jwt_token(ctx)
     return token
 end
 
+-- To get new access token by calling get token api
+local function generate_token_using_password_grant(conf,ctx)
+    log.warn("generate_token_using_password_grant Function Called")
+    --Read Body
+    ngx.req.read_body()
+    --Get Body Data
+    local request_body=ngx.req.get_body_data()
```

Comment:
You can use `core.request.get_body()` instead.

Reply:
Done. I changed it.